ci-operator/step-registry/ibmcloud/provision/vpc/ibmcloud-provision-vpc-commands.sh

```diff
@@ -60,6 +60,10 @@ function ibmcloud_login {
     "${IBMCLOUD_CLI}" config --check-version=false
     echo "Try to login..." 
     "${IBMCLOUD_CLI}" login -r ${region} --apikey @"${CLUSTER_PROFILE_DIR}/ibmcloud-api-key"
+    "${IBMCLOUD_CLI}" version
+    "${IBMCLOUD_CLI}" plugin list
+    "${IBMCLOUD_CLI}" plugin update -all -f
```

Comment:
This bring some risk that some unexpected plugin update break the existing ci, if you want to update plugin, supposed to do that in upi-installer image where plugin is being installed.